Editor/AGS.Editor/Utils/PaletteUtilities.cs

```diff
@@ -233,20 +232,20 @@ private static void RemapPixels(byte[] pixels, Color[] oldPal, Color[] newPal, b
         }
 
         // 1.5k lookup table for color matching
-        private static uint[] ColDiffTable = new uint[3 * 128];
+        private static uint[] ColDiffTable = new uint[3 * 512];
```

Comment:
The above comment does not match the table now, please fix it, or remove the comment.